dataset_eval/evaluate.py

```diff
@@ -23,6 +23,7 @@
 import re
 
 SLEEP_DURATION = 30  # number of seconds to wait between runs
+HISTORY_STORAGE_DIR = os.path.join(config.FILE_STORAGE_DIR, "history")
```

Comment:
does it make sense to put the existing high level files here too? what do you think?